pkg/build/pipelines/ruby/install.yaml

```diff
@@ -0,0 +1,20 @@
+name: Run a gem install
+
+needs:
+  packages:
+    - busybox
+    - ca-certificates-bundle
```

Comment:
Does this pipeline need to depend on a package that provides `gem`?

Reply:
In other words, I think this should depend on `ruby-3.2` so that it can know it has `gem` available. But this leads to a couple questions:

1. should it depend on `ruby-3.2` or `ruby-3.1`, or just anything that says it `provides: ruby`?
2. `ruby-3.2` isn't a package name in Alpine-land, so this package will be distro-specific if we name it `ruby-3.2`. This leads to a sort of general question about whether/how pipelines can depend on distro-agnostic package names, or distro-specific package names.

Reply:
We might be able to depend on `cmd:gem`.

Reply:
I was going to ask, but the test I was using already required ruby.

Reply:
@imjasonh @kaniini I'd love to get this merged in so I can use exercise it for all the ruby work I'm doing. Is this

> We might be able to depend on cmd:gem.

a possibility right now? Otherwise what do you think about leaving the ruby dependency off for now? Then in the future we could do something like the `go` pipelines? Looks like they just depend on the latest version of `go`, or at least whatever is published under that package

Reply:
I added the which ruby test to force a user to have ruby installed.

Reply:
We can just require the user provide a ruby for now, yes.